src/dsp_tools/dsp_tools.py

```diff
@@ -88,34 +100,52 @@ def make_parser() -> argparse.ArgumentParser:
-                               help="URL of the SIPI server where DSP-TOOLS sends the multimedia files to")
-    parser_upload.add_argument("-i", "--imgdir", default=".", help="folder from where the paths in the <bitstream> tags are evaluated")
-    parser_upload.add_argument("-I", "--incremental", action="store_true",
-                               help="The links in the XML file point to IRIs (on the server) instead of IDs (in the same XML file).")
-    parser_upload.add_argument("-V", "--validate", action="store_true", help="validate the XML file without uploading it")
+    parser_upload.add_argument(
+        "-S",
+        "--sipi",
+        default=default_sipi,
+        help="URL of the SIPI server where DSP-TOOLS sends the multimedia files to",
+    )
+    parser_upload.add_argument(
+        "-i", "--imgdir", default=".", help="folder from where the paths in the <bitstream> tags are evaluated"
+    )
+    parser_upload.add_argument(
+        "-I",
+        "--incremental",
+        action="store_true",
+        help="The links in the XML file point to IRIs (on the server) instead of IDs (in the same XML file).",
+    )
+    parser_upload.add_argument(
+        "-V", "--validate", action="store_true", help="validate the XML file without uploading it"
+    )
     parser_upload.add_argument("-v", "--verbose", action="store_true", help=verbose_text)
     parser_upload.add_argument("-m", "--metrics", action="store_true", help="write metrics into a 'metrics' folder")
     parser_upload.add_argument("xmlfile", help="path to the XML file containing the data")
 
     # process-files
     parser_process_files = subparsers.add_parser(
         name="process-files",
-        help="For internal use only: process all files referenced in an XML file"
+        help="For internal use only: process all files referenced in an XML file",
     )
     parser_process_files.set_defaults(action="process-files")
-    parser_process_files.add_argument("--input-dir", help="path to the input directory where the files should be read from")
-    parser_process_files.add_argument("--output-dir", help="path to the output directory where the processed/transformed files should be written to")
+    parser_process_files.add_argument(
+        "--input-dir", help="path to the input directory where the files should be read from"
+    )
+    parser_process_files.add_argument(
+        "--output-dir", help="path to the output directory where the processed/transformed files should be written to"
+    )
     parser_process_files.add_argument("--nthreads", type=int, default=None, help="number of threads to use")
     parser_process_files.add_argument("xml_file", help="path to XML file containing the data")
 
     # upload-files
     parser_upload_files = subparsers.add_parser(
         name="upload-files",
-        help="For internal use only: upload already processed files"
+        help="For internal use only: upload already processed files",
```

Comment:
It seems weird at first glance, but after thinking more deeply about it, I began to find it cool. It reduces git diffs: If you add a new parameter, the diff will only highlight the single line that you added, because the comma on the old line is already there.

But I agree that black is opinionated (they characterize themselves as opinionated, but my opinion is very close to theirs, so I'm super happy 😄 )